src/ocean/util/container/Container.d

```diff
@@ -320,7 +322,7 @@ struct Container
 
                 void collect (T* p)
                 {
-                        assert (p);
+                        verify(p);
```

Comment:
You may want `p !is null` here, like in line 165. (and it matches the preference of @david-eckardt-sociomantic )

Reply:
Huh, not sure how this slipped through.